Podcast/PlayerViewController.swift

```diff
@@ -204,7 +251,34 @@ class PlayerViewController: TabBarAccessoryViewController, PlayerDelegate, Playe
     }
     
     func playerControlsDidTapMoreButton() {
-        
+        let likeOption = ActionSheetOption(title: "Like this episode", titleColor: .charcoalGrey, image: #imageLiteral(resourceName: "heart_icon")) {
```

Comment:
we will be abstracting all of this away shortly, to much repeated code but its okay. Good for now. It's a problem in almost all our controllers